index.js

```diff
@@ -3,12 +3,13 @@ var cli = require('./libs/cli');
 
 program
   .version('0.0.1')
+  .option('-d, --directions', 'Repeat the current challenge directions', cli.directions)
   .option('-c, --check', 'Check your current work', cli.check)
   .option('-r, --reset <level>', 'reset the current level, or to a previous level', cli.reset)
   .option('-n, --hint', 'Get a hint', cli.hint)
   .option('-p, --play', 'Get directions, and play the game! This is the default command when you run `gitfun` with no arguments.', cli.main)
   .parse(process.argv);
 
-if(process.argv.length === 2) { 
+if(process.argv.length === 2) {
   cli.main();
-}
\ No newline at end of file
+}
```

Comment:
My editor is set up to trim trailing whitespace on save and add a newline to EOF if not present. I consider it good practice but it generates some annoying diffs. Sorry for the noise.